code/__DEFINES/access.dm

```diff
@@ -150,3 +150,207 @@
+#define WILDCARD_FLAG_PRV_COMMAND	ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND
+/// Name associated with the private command wildcard bitflag.
+#define WILDCARD_NAME_PRV_COMMAND	ACCESS_FLAG_PRV_COMMAND_NAME
+/// Access flags that can be applied to captain wildcard slots.
+#define WILDCARD_FLAG_CAPTAIN		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN
+/// Name associated with the captain wildcard bitflag.
+#define WILDCARD_NAME_CAPTAIN		ACCESS_FLAG_CAPTAIN_NAME
+/// Access flags that can be applied to centcom wildcard slots.
+#define WILDCARD_FLAG_CENTCOM		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_CENTCOM
+/// Name associated with the centcom wildcard bitflag.
+#define WILDCARD_NAME_CENTCOM		ACCESS_FLAG_CENTCOM_NAME
+/// Access flags that can be applied to syndicate wildcard slots.
+#define WILDCARD_FLAG_SYNDICATE		ACCESS_FLAG_COMMON | ACCESS_FLAG_COMMAND | ACCESS_FLAG_PRV_COMMAND | ACCESS_FLAG_CAPTAIN | ACCESS_FLAG_SYNDICATE
+/// Name associated with the syndicate wildcard bitflag.
+#define WILDCARD_NAME_SYNDICATE		ACCESS_FLAG_SYNDICATE_NAME
+/// Access flags that can be applied to offstation wildcard slots.
+#define WILDCARD_FLAG_AWAY			ACCESS_FLAG_AWAY
+/// Name associated with the offstation wildcard bitflag.
+#define WILDCARD_NAME_AWAY			ACCESS_FLAG_AWAY_NAME
+/// Access flags that can be applied to super special weird wildcard slots.
+#define WILDCARD_FLAG_SPECIAL		ACCESS_FLAG_SPECIAL
+/// Name associated with the super special weird wildcard bitflag.
+#define WILDCARD_NAME_SPECIAL		ACCESS_FLAG_SPECIAL_NAME
+/// Access flag that indicates a wildcard was forced onto an ID card.
+#define WILDCARD_FLAG_FORCED		ALL
+/// Name associated with the wildcard bitflag that covers wildcards that have been forced onto an ID card that could not accept them.
+#define WILDCARD_NAME_FORCED		"Hacked"
+
+/// Departmental/general/common area accesses.
+#define COMMON_ACCESS 			list(ACCESS_MECH_MINING, ACCESS_MECH_MEDICAL, ACCESS_MECH_SECURITY, ACCESS_MECH_SCIENCE, \
+								ACCESS_MECH_ENGINE, ACCESS_AUX_BASE, ACCESS_PSYCHOLOGY, ACCESS_PHARMACY, ACCESS_NETWORK, \
+								ACCESS_WEAPONS, ACCESS_MINERAL_STOREROOM, ACCESS_SEC_DOORS, ACCESS_XENOBIOLOGY, \
+								ACCESS_MINING_STATION, ACCESS_MAILSORTING, ACCESS_MINING, ACCESS_RESEARCH, ACCESS_THEATRE, \
+								ACCESS_SURGERY, ACCESS_COURT, ACCESS_QM, ACCESS_VIROLOGY, ACCESS_LAWYER, ACCESS_LIBRARY, \
+								ACCESS_HYDROPONICS, ACCESS_CHEMISTRY, ACCESS_CONSTRUCTION, ACCESS_CARGO, ACCESS_ROBOTICS, \
+								ACCESS_KITCHEN, ACCESS_CREMATORIUM, ACCESS_JANITOR, ACCESS_BAR, ACCESS_CHAPEL_OFFICE, \
+								ACCESS_EXTERNAL_AIRLOCKS, ACCESS_MAINT_TUNNELS, ACCESS_ENGINE_EQUIP, ACCESS_ENGINE, \
+								ACCESS_GENETICS, ACCESS_RND, ACCESS_MORGUE, ACCESS_MEDICAL, ACCESS_FORENSICS_LOCKERS, \
+								ACCESS_BRIG, ACCESS_SECURITY)
```

Comment:
```suggestion
#define COMMON_ACCESS list(\
	ACCESS_MECH_MINING, \
	ACCESS_MECH_MEDICAL, \
	ACCESS_MECH_SECURITY, \
	ACCESS_MECH_SCIENCE, \
	ACCESS_MECH_ENGINE, \
	ACCESS_AUX_BASE, \
	ACCESS_PSYCHOLOGY, \
	ACCESS_PHARMACY, \
	ACCESS_NETWORK, \
	ACCESS_WEAPONS, \
	ACCESS_MINERAL_STOREROOM, \
	ACCESS_SEC_DOORS, \
	ACCESS_XENOBIOLOGY, \
	ACCESS_MINING_STATION, \
	ACCESS_MAILSORTING, \
	ACCESS_MINING, \
	ACCESS_RESEARCH, \
	ACCESS_THEATRE, \
	ACCESS_SURGERY, \
	ACCESS_COURT, \
	ACCESS_QM, \
	ACCESS_VIROLOGY, \
	ACCESS_LAWYER, \
	ACCESS_LIBRARY, \
	ACCESS_HYDROPONICS, \
	ACCESS_CHEMISTRY, \
	ACCESS_CONSTRUCTION, \
	ACCESS_CARGO, \
	ACCESS_ROBOTICS, \
	ACCESS_KITCHEN, \
	ACCESS_CREMATORIUM, \
	ACCESS_JANITOR, \
	ACCESS_BAR, \
	ACCESS_CHAPEL_OFFICE, \
	ACCESS_EXTERNAL_AIRLOCKS, \
	ACCESS_MAINT_TUNNELS, \
	ACCESS_ENGINE_EQUIP, \
	ACCESS_ENGINE, \
	ACCESS_GENETICS, \
	ACCESS_RND, \
	ACCESS_MORGUE, \
	ACCESS_MEDICAL, \
	ACCESS_FORENSICS_LOCKERS, \
	ACCESS_BRIG, \
	ACCESS_SECURITY \
	)
```
Bit more readable and less likely to cause conflicts if two PRs touch the same code.

Reply:
Within scope. I'm moving these to be cached in SSid_access. The less hardcoded stuff like this the better. More flexible. The intent of the new system is to be as flexible as possible. Station traits can change them. Admins can VV change them. So on, so forth.

Reply:
Okay. Cached in SSid_access now. They weren't used in a lot of places of code, outside of that subsystem to begin with, to be fair.